waltz-ng/client/auth-sources/components/non-auth-sources-list.html

```diff
@@ -0,0 +1,68 @@
+<div class="row">
+    <div class="col-md-9">
+        <table class="table table-condensed">
+            <thead>
+            <tr>
+                <th>Application</th>
+                <th>Data Type</th>
+                <th>Consumers</th>
+            </tr>
+            </thead>
+            <tbody >
+            <tr ng-repeat="source in $ctrl.nonAuthSources">
+                <td>
+                    <a ng-bind="source.name"
+                       ui-sref="main.app.view( { id: source.id })">
+                    </a>
+                </td>
+                <td>
+                    <a ng-bind="source.dataType.code | toDisplayName:'dataType'"
+                       ui-sref="main.data-type.code ({ code: source.dataType.code })">
+                    </a>
+                </td>
+                <td>
+                    <a ng-click="$ctrl.showDetail(source)" class="clickable">
+                        <span class="label"
+                              ng-class="{
+                                'label-danger':source.consumers.length > 0,
+                                'label-success':source.consumers.length == 0
+                                }"
+                              ng-bind="source.consumers.length">
+                        </span>
+                    </a>
+
+                </td>
+            </tr>
+            </tbody>
+        </table>
+    </div>
+    <div class="col-md-3">
+        <div ng-if="$ctrl.selected != null">
+            <h5>Detail</h5>
+            <p class="small">
+                Application
+                <mark ng-bind="$ctrl.selected.name"></mark>
+                has not been declared Authoritative by any organisational unit.
+            </p>
+
+            <h5>Consumers</h5>
+            <div class="waltz-scroll-region" style="height:200px">
```

Comment:
missed opportunity for using whizzy new scroll-region!  (perhaps combined with the `ng-class` trick)